### PR TITLE
Pass in configured TelemetryConfiguration to LoggingSetup in JobRunner

### PR DESCRIPTION
**Pull request by xavierdecoster**

### CODE REVIEW

File: src/NuGet.Jobs.Common/JobRunner.cs [hunk 1]
```diff
@@ -194,9 +194,12 @@ private static ApplicationInsightsConfiguration ConfigureApplicationInsights(Job
             return applicationInsightsConfiguration;
         }
 
-        private static ILoggerFactory ConfigureLogging(JobBase job)
+        private static ILoggerFactory ConfigureLogging(JobBase job, TelemetryConfiguration telemetryConfiguration = null)
```
Comment (joelverhagen) [inline src/NuGet.Jobs.Common/JobRunner.cs]: Why a default parameter on a private method? Might ease code readability to name an explicit null parameter.

File: src/NuGet.Jobs.Common/JobRunner.cs [hunk 2]
```diff
@@ -194,9 +194,12 @@ private static ApplicationInsightsConfiguration ConfigureApplicationInsights(Job
...
         {
-            var loggerFactory = LoggingSetup.CreateLoggerFactory(LoggingSetup.CreateDefaultLoggerConfiguration(true));
+            var loggerFactory = LoggingSetup.CreateLoggerFactory(
+                LoggingSetup.CreateDefaultLoggerConfiguration(true),
+                telemetryConfiguration: telemetryConfiguration);
```
Comment (joelverhagen) [inline src/NuGet.Jobs.Common/JobRunner.cs]: nit: unnecessary named parameter

File: src/NuGet.Jobs.Common/JobRunner.cs (hunk 1 above)
Comment (xavierdecoster) [reply src/NuGet.Jobs.Common/JobRunner.cs]: Fixed in https://github.com/NuGet/NuGet.Jobs/pull/840/commits/084036dbca2738da43c5e352f091c93614987c25

File: src/NuGet.Jobs.Common/JobRunner.cs (hunk 2 above)
Comment (xavierdecoster) [reply src/NuGet.Jobs.Common/JobRunner.cs]: It is necessary as compiler complains about it :) There are other optional arguments skipped in this call.

**Pull request merged**